coa-helm/templates/identity.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: {{ .Values.identity.name }}
+spec:
+  type: 0
+  resourceID: {{ .Values.identity.resourceId }}
+  clientID: {{ .Values.identity.clientId }}
```

Comment:
I do not understand why Azure can't be satisfied with a single value for identifying identities 🙄

Reply:
Lol I know right